app/views/layouts/application.html.haml

```diff
@@ -52,6 +52,10 @@
                 - current_site.posts.published.order_by_recently.limit(5).each do |post|
                   = render partial: 'posts/summary_with_content', locals: {post: post}
 
+            - if current_site.serials.present?
+              %h2.subheading.subheading--aside.subheading--press-articles 連載
+              = link_to 'すべての連載を見る', serials_path(current_site)
```

Comment:
Thanks! I've fixed it. How about https://github.com/bm-sms/daimon-news/commit/4af39a42c9875c545b163c797eeb1e2b01bebeb9 ?